wordpress-mobile/WordPress-Android
# Fix Quick Start bottom padding on the My Site screen

Fixes #15430

https://user-images.githubusercontent.com/1405144/143029388-c3699106-7989-4895-b95b-5d0c80b09c63.mp4

Cc @osullivanchris 

To test:

- Follow the Quick Start steps after creating a new site.
- Tap the “Visit your site” step in the Quick Start so that a snack bar gets displayed.
- Wait for the snackbar to auto-dismiss.
- Notice that the blank area occupied by the snackbar (that was shown earlier even after snackbar dismiss) is now not shown (see timestamp `0.06` in the video).

Review Instructions:

Only 1 reviewer is sufficient.

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
